l10n_br_account/models/account_invoice_line.py

```diff
@@ -212,6 +212,10 @@ def create(self, vals_list):
             )
 
         lines = super().create(vals_list)
+        # Hack to call again the onchange subtotal to force to use price_total instead
+        for line in lines:
+            line._get_fields_onchange_subtotal()
+
         if dummy_doc.id != fiscal_doc_id:
```

Comment:
@felipemotter montei uma branch com as PRs que temos e aqui tive um problema na hora de Confirmar a invoice

---> local variable 'fiscal_doc_id' referenced before assignment

para contornar eu adicionei isso antes de for values in vals_list (linha 198)

`        fiscal_doc_id = (self.env["account.move"])`

Reply:
eu não estou conseguindo replicar esse erro, consegues me dar mais detalhes?

Reply:
@felipemotter Vou testar novamente e te falo.